content/minfraud/release-notes/2021.mdx

```diff
@@ -28,74 +29,83 @@ Private Relay works and delivers the best possible user experience for your
-* A new value for the [`/disposition/action` output](/minfraud/api-documentation/responses#schema--response--disposition__action),
-“test”. This additional disposition action can be used to separate transactions
-for rules which you are interested in actively testing without affecting your
-existing workflows.
-
-For more information, see our [blog post](https://blog.maxmind.com/2021/09/new-minfraud-features-passing-3-d-secure-outcome-custom-rule-label-in-minfraud-response-test-disposition-for-custom-rules/).
+- A new input,
+  [`/credit_card/was_3d_secure_successful`](/minfraud/api-documentation/requests#schema--request--credit-card__was_3d_secure_successful),
+  which allows you to send us whether the outcome of
+  [3-D Secure verification](https://en.wikipedia.org/wiki/3-D_Secure) was
+  successful. This can help us improve your risk scoring.
+- A new output,
+  [`/disposition/rule_label`](/minfraud/api-documentation/responses#schema--response--disposition__rule_label),
+  which returns the label of the custom rule that affected a transaction.
+- A new value for the
+  [`/disposition/action` output](/minfraud/api-documentation/responses#schema--response--disposition__action),
+  “test”. This additional disposition action can be used to separate
+  transactions for rules which you are interested in actively testing without
+  affecting your existing workflows.
+
+For more information, see our
+[blog post](https://blog.maxmind.com/2021/09/new-minfraud-features-passing-3-d-secure-outcome-custom-rule-label-in-minfraud-response-test-disposition-for-custom-rules/).
 
 Our [client APIs](/minfraud/evaluate-a-transaction#links-to-maxmind-client-apis)
 have been updated to support these outputs so may need to refresh yours if you
 are not interfacing directly with our REST API.
+
 </ReleaseNote>
 
 <ReleaseNote date="2021-05-19" title="Updated warning codes in minFraud Score, Insights, and Factors">
 We updated warning codes for [minFraud Score, Insights and Factors](/minfraud)
 responses in the [/warnings/
 object](/minfraud/api-documentation/responses/#schema--response--warnings).
 
-* We added `BILLING_REGION_NOT_FOUND` and `SHIPPING_REGION_NOT_FOUND` codes
-* We updated the warning explanations for certain warning codes to include that
-distance calculations (outlined below) may be impacted when certain location
-information is missing or cannot be found
-    - `/shipping_address/distance_to_ip_location`
-    - `/shipping_address/distance_to_billing_address`
-    - `/billing_address/distance_to_ip_location`
+- We added `BILLING_REGION_NOT_FOUND` and `SHIPPING_REGION_NOT_FOUND` codes
+- We updated the warning explanations for certain warning codes to include that
+  distance calculations (outlined below) may be impacted when certain location
+  information is missing or cannot be found -
+  `/shipping_address/distance_to_ip_location` -
+  `/shipping_address/distance_to_billing_address` -
+  `/billing_address/distance_to_ip_location`
```

Comment:
Similar issue here: used to be a list, but now it is not. Should probably be changed back to a list.